week-1-practise/level-1/8-function-output-1.js

```diff
@@ -7,6 +7,10 @@ console.log(result);
 
 // Answer these questions:
 // - What does this program do?
+    /*it generates random numbers between 0 and 9; includes 0 but not 10*/ 
```

Comment:
Actually this will return any floating-point number between 0 and 10, not including 10. So for example, it could generate 9.99

Reply:
I was thinking of Math.floor((Math.random() * 10) );